src/Workspaces/Core/MSBuild/MSBuild/ProjectFile/CommandLineArgumentReader.cs

```diff
@@ -168,7 +168,8 @@ protected void ReadDebugInfo()
             {
                 var debugType = Project.ReadPropertyString(PropertyNames.DebugType);
 
-                if (s_debugTypeValues.TryGetValue(debugType, out var value))
+                if (!string.IsNullOrWhiteSpace(debugType) &&
+                    s_debugTypeValues.TryGetValue(debugType, out var value))
```

Comment:
Is fixing the same issue as #30209 which is currently in flight from a community member?

Reply:
Ah, yes -- I hadn't noticed that. Adding a test for the repro project in #29122 exposed the same issue. So, I needed to fix it as part of that.

Reply:
I left my review on #30209. Feel free to merge that. It's a righteous change. Once it's in master, I'll rebase this PR.